### DISCUSSION TITLE
Backend

### DISCUSSION THREAD

**Discussion by myOmikron** [hopfenspace/hackaburg-2023#5] [Ideas]
foo

**Comment by WebFreak001** [comment]
- database
	- collection points
	- shops
	- articles/items of shops
	- drivers (+ currently routed destination)
	- active orders
		- estimated driving time + shop provision time
	- fulfilled orders
	- users/accounts
	- shopping carts
	- driver work schedules
	- recurring orders / subscriptions / planned deliveries
- time table / current schedule where which drivers are supposed to be at which point in time
- product search (suggesting shops based on shopping cart)
